src/shortcuts/ShortcutUtils.js

```diff
@@ -1,25 +1,40 @@
 import Lang from 'lodash';
 import moment from 'moment';
 
-export function createSentenceFromStructuredData(structuredPhraseTemplate, getAttributeValue, textIfNoData) {
-    let last = 0, valueName, value;
-    let start = structuredPhraseTemplate.indexOf("${"), end;
-    let result = "";
+// This function is similar to createSentenceFromStructuredData except it adds spans around the structured data so that placeholders are styled appropriately
```

Comment:
It looks like the last parameter `isSigned` is optional. Do we want to default it here?

Reply:
done